packages/wrangler/src/pages/dev.tsx

```diff
@@ -78,6 +79,11 @@ export function Options(yargs: Argv) {
 				description:
 					"The location of the single Worker script if not using functions",
 			},
+			"bundle-worker": {
```

Comment:
We _will_ make this default at some point pretty soon. So I'm tempted to make this an env var or deprecated/experimental or whatever so we can remove it easily when we force it. What do you think, @petebacondarwin ?

Reply:
Actually, to keep in line with `wrangler publish` we should keep this and call it `bundle`. The usage in `wrangler publish` is that this defaults to true but can be turned off with `no-bundle` or `bundle=false`.

So I would rather not hide it away - I think there may be valid cases where people choose not to run the bundler on the generated `_worker.js`.